projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -354,7 +354,7 @@ function wpcom_launchpad_get_task_definitions() {
 				if ( ( new Automattic\Jetpack\Status\Host() )->is_atomic_platform() ) {
 					return admin_url( 'admin.php?page=jetpack#/discussion' );
```

Comment:
@mashikag, will we also need the URL to be updated for Atomic sites? If so, what's the best way to detect whether we should use the new URL or not?

Reply:
Why do we redirect to Jetpack settings pages on Atomic? Those pages are missing most of the Newsletter settings, and are designed for self hosted Jetpack plugin customers, not .com customers, so it doesn't make sense.

Let's keep things consistent link all .com customers to .com pages.

Reply:
As per @simison's point, I will update the URL to be the same for all users, incl. Jetpack self-hosted sites.

Reply:
>  will update the URL to be the same for all users, incl. Jetpack self-hosted sites.


@mashikag Jetpack self hosted sites don't get this launchpad UI at all.

Reply:
@simison, thanks for the correction. I meant to say "incl. Atomic sites". 🙇